app/queues/reconciler/reconciler-queue.server.ts

```diff
@@ -0,0 +1,44 @@
+import logger from '~/lib/logger.server';
+import { reconcilerQueue } from './reconciler-worker.server';
+
+declare global {
+  var __reconciler_queue_init__: boolean;
+}
+
+reconcilerQueue.on('error', (err) => {
+  logger.error('Reconciler encountered an error', err);
+});
+
+// function to add jobs
+export const addReconcilerJob = async () => {
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
Can you explain this to me?  If we're in development, you're not going to run this?

Reply:
yep, that's a bug